_gcode/M420.md

```diff
@@ -12,7 +12,7 @@ codes:
   - M420
 
 long:
-  - Get and/or set the enabled state of bed leveling compensation, plus Z fade height.
+  - Get and/or set the enabled state of bed leveling compensation, plus Z fade height. Units will be in mm unless *inch* mode is set with [`G20`](/docs/gcode/G020.html) (which requires `INCH_MODE_SUPPORT`).
```

Comment:
You think we should add this kind of note to every linear unit parameter?
A better idea is not to mention mm or inches. Just say "in current units."